### PR TITLE
LPS-205028 Fix error message on form container when guest user tries to add object entry beyond their limit

### PR DESCRIPTION
**Pull request by thallescezar**
Related issue: https://liferay.atlassian.net/browse/LPS-205028

### CODE REVIEW

**Comment by thallescezar** [comment]
ci:test:sf

File: modules/apps/info/info-api/src/main/java/com/liferay/info/exception/InfoFormValidationException.java [hunk 1]
```diff
@@ -57,6 +57,29 @@ public String getLocalizedMessage(Locale locale) {
 
 	}
 
+	public static class ExceedsMaxEntries extends InfoFormValidationException {
+
+		public ExceedsMaxEntries(String messageKey, String label) {
```
Comment (gabrielwas) [inline modules/apps/info/info-api/src/main/java/com/liferay/info/exception/InfoFormValidationException.java:62]: Rename it to infoFormLabel, please

**Review by gabrielwas — CHANGES_REQUESTED** [review]
Check the comment, please

**Comment by thallescezar** [comment]
ci:test:object

**Comment by thallescezar** [comment]
ci:test:relevant

**Review by gabrielwas — APPROVED** [review]
LGTM :)
Please, open a PR to the Echo Team so they can check the changes in the info module.

**Comment by thallescezar** [comment]
Resent here: https://github.com/liferay-echo/liferay-portal/pull/15019

**Pull request closed without merging**